KakaoImageSearch/Api/Model/ImageItem.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ImageItem.swift
+//  KakaoImageSearch
+//
+//  Created by 강현석 on 2021/01/06.
+//
+
+import Foundation
+
+struct ImageItem: Codable {
+    let collection: String
+    let thumbnailUrl: String
```

Comment:
위에 설명했던 약어와 같은 이슈겠네요 👍 

```suggestion
    let thumbnailURL: String
```

Reply:
감사합니다 👍 
수정하였습니다 [576b6bb]